api-reference/10 UI Components/dxHtmlEditor/3 Methods/getSelection().md

```diff
@@ -15,6 +15,6 @@ A zero-based index at which the selection starts.
  [Embedded items](/concepts/05%20UI%20Components/HtmlEditor/10%20Formats '/Documentation/Guide/UI_Components/HtmlEditor/Formats/') have a length of 1.
 
```

Comment:
Since I'm unable to edit Line 9, can you please make the following change?

`The selected content's range. It has the following structure:`